src/quotamodel.c

```diff
@@ -239,7 +239,7 @@ static bool get_table_size_entry_flag(TableSizeEntry *entry, TableSizeEntryFlag
 static void reset_table_size_entry_flag(TableSizeEntry *entry, TableSizeEntryFlag flag);
 static void set_table_size_entry_flag(TableSizeEntry *entry, TableSizeEntryFlag flag);
 
-static void delete_from_table_size_map(char *str);
+static void delete_from_table_size_map(ArrayBuildState *tableids, ArrayBuildState *segids);
```

Comment:
It is not necessary to add prototypes for static functions. [I added a prototype](https://github.com/arenadata/diskquota/commit/14b861def6c8ef018c522ceff67110823d52e0ff#diff-74478810938875c96024185d7ae83994b0aa5aaf71f9398ef87a03d5dfd2d9d7R225) for deletion because the first use of the function came before the definition.